build.gradle

```diff
@@ -171,3 +172,20 @@ shadowJar {
 
 build.dependsOn shadowJar
 jar.enabled = false
+
+modrinth {
+    token = System.getenv("MODRINTH_TOKEN")
+    projectId = "multiverse-core"
+    versionNumber = version
+
+    if(System.getenv("PRERELEASE")) {
+        versionType = "beta"
+    } else {
+        versionType = "release"
+    }
+
+    uploadFile = jar
+    gameVersions = ["1.19.2", "1.19.1", "1.19", "1.18.2", "1.18.1", "1.18", "1.17.1", "1.17", "1.16.5", "1.16.4", "1.16.3", "1.16.2", "1.16.1", "1.16"]
+    loaders = ["bukkit", "spigot", "paper", "purpur"]
```

Comment:
not very sure if we should explicitly put purpur hm

Reply:
Welp, It does support it, so I don't really see why not